README.md

```diff
@@ -213,6 +217,16 @@ CDC including this GitHub page may be subject to applicable federal law, includi
 
 #### ReportStream Setup
 
+### Generating and using a token
+1. Run `brew install mike-engel/jwt-cli/jwt-cli`
+2. Run `jwt encode --exp='+5min' --jti $(uuidgen) --alg RS256  --no-iat -S @/PATH_TO_FILE_ON_YOUR_MACHINE/trusted-intermediary/mock_credentials/organization-trusted-intermediary-private-key-local.pem`
+3. Copy token from terminal and paste into your postman body with the key `client_assertion`
+4. Add a key to the body with the key `scope` and value of `trusted-intermediary`
+5. Body type should be `x-wwww-form-urlencoded`
+6. You should be able to run the post call against the `v1/auth/token` endpoint to receive a bearer token [to be used in this step](#submit-request-to-reportstream)
```

Comment:
I just realized that this description is for calling us, not ReportStream.  This section probably goes best under the `Using and Running` heading.